lib/apiservers/engine/backends/container.go

```diff
@@ -1377,14 +1377,22 @@ func validateCreateConfig(config *types.ContainerCreateConfig) error {
 	config.HostConfig.Memory = memoryMB
 	log.Infof("Container memory: %d MB", config.HostConfig.Memory)
 
-	if config.NetworkingConfig == nil {
-		config.NetworkingConfig = &dnetwork.NetworkingConfig{}
-	}
-
 	if config.HostConfig == nil || config.Config == nil {
 		return BadRequestError("invalid config")
 	}
 
+	if config.NetworkingConfig == nil {
+		config.NetworkingConfig = &dnetwork.NetworkingConfig{}
+	} else {
+		if l := len(config.NetworkingConfig.EndpointsConfig); l > 1 {
+			return fmt.Errorf("Error setting NetWorkMode: Container cannot be connected to more than one network endpoints; current endpoints: %d", l)
```

Comment:
"NetworkMode error: Container can be connected to one network endpoint only" Number of current endpoints doesn't look as an interesting data.